Modules/_ssl.c

```diff
@@ -5626,12 +5624,25 @@ _ssl_enum_certificates_impl(PyObject *module, const char *store_name)
     PCCERT_CONTEXT pCertCtx = NULL;
     PyObject *keyusage = NULL, *cert = NULL, *enc = NULL, *tup = NULL;
     PyObject *result = NULL;
+    static DWORD system_stores[] = {
```

Comment:
I don't like code duplication. Please make the system store list a global static